draft-ietf-mops-streaming-opcons.md

```diff
@@ -501,46 +479,33 @@ It is worth noting that not all high-demand content is "live" content. One relev
-
-In his talk, Sanjay Mishra {{Mishra}} reported that after the CoViD-19 pandemic broke out in early 2020,
+* BitTorrent also used distributed hash tables to organize peers into a ring topology, where each peer knew its "next peer" and "previous peer". There was no connection between the application-level ring topology and the lower-level network topology, so a peer's "next peer" might be anywhere on the reachable Internet. Traffic models that expected most communication to take place with a relatively small number of servers were unable to cope with peer-to-peer traffic that was much less predictable.
 
-- Comcast's streaming and web video consumption rose by 38%, with their reported peak traffic up 32% overall between March 1 to March 30,
-- AT&T reported a 28% jump in core network traffic (single day in April, as compared to pre stay-at-home daily average traffic), with video accounting for nearly half of all mobile network traffic, while
-social networking and web browsing remained the highest percentage (almost a quarter each) of overall mobility traffic, and
-- Verizon reported similar trends with video traffic up 36% over an average day (pre COVID-19)}.
-
-We note that other operators saw similar spikes during this time period. Craig Labowitz {{Labovitz}} reported
+Especially as end users increase use of video-based social networking applications, it will be helpful for access network providers to watch for increasing numbers of end users uploading significant amounts of content.
 
-- Weekday peak traffic increases over 45%-50% from pre-lockdown levels,
-- A 30% increase in upstream traffic over their pre-pandemic levels, and
-- A steady increase in the overall volume of DDoS traffic, with amounts exceeding the pre-pandemic levels by 40%. (He attributed this increase to the significant rise in gaming-related DDoS attacks ({{LabovitzDDoS}}), as gaming usage also increased.)
+### Impact of Global Pandemic {{IABcovid}}
 
-Subsequently, the Internet Architecture Board (IAB) held a COVID-19 Network Impacts Workshop {{IABcovid}} in November 2020. Given a larger number of reports and more time to reflect, the following observations from the draft workshop report are worth considering.
+Early in 2020, the CoViD-19 pandemic and resulting quarentines and shutdowns led to significant changes in traffic patterns, resulting from a causes such as large number of people who were suddenly working and attending school remotely, and using more interactive applications (video conderencing, in addition to streaming media). Subsequently, the Internet Architecture Board (IAB) held a COVID-19 Network Impacts Workshop {{IABcovid}} in November 2020. The following observations from the workshop report are worth considering.
```

Comment:
s/Early in 2020, the CoViD-19 pandemic and resulting quarentines and shutdowns led to significant changes in traffic patterns, resulting from a causes such as large number of people who were suddenly working and attending school remotely, and using more interactive applications (video conderencing, in addition to streaming media). Subsequently, the Internet Architecture Board (IAB) held a COVID-19 Network Impacts Workshop {{IABcovid}} in November 2020. The following observations from the workshop report are worth considering.
/Early in 2020, the CoViD-19 pandemic and resulting quarantines and shutdowns led to significant changes in traffic patterns, due to a large number of people who suddenly started working and attending school remotely and using more interactive applications (video conferencing, in addition to streaming media). Subsequently, the Internet Architecture Board (IAB) held a COVID-19 Network Impacts Workshop {{IABcovid}} in November 2020. The following observations from the workshop report are worth considering.